keyboards/masterworks/classy_tkl/info.json

```
@@ -0,0 +1,292 @@
+                { "label": "!", "x": 1, "y": 1.5 },
+                { "label": "\"", "x": 2, "y": 1.5 },
+                { "label": "\u00a3", "x": 3, "y": 1.5 },
+                { "label": "$", "x": 4, "y": 1.5 },
+                { "label": "%", "x": 5, "y": 1.5 },
+                { "label": "^", "x": 6, "y": 1.5 },
+                { "label": "&", "x": 7, "y": 1.5 },
+                { "label": "*", "x": 8, "y": 1.5 },
+                { "label": "(", "x": 9, "y": 1.5 },
+                { "label": ")", "x": 10, "y": 1.5 },
+                { "label": "_", "x": 11, "y": 1.5 },
+                { "label": "+", "x": 12, "y": 1.5 },
+                { "label": "Backspace", "x": 13, "y": 1.5, "w": 2 },
+                { "label": "Insert", "x": 15.25, "y": 1.5 },
+                { "label": "Home", "x": 16.25, "y": 1.5 },
+                { "label": "PgUp", "x": 17.25, "y": 1.5 },
+                { "label": "Tab", "x": 0, "y": 2.5, "w": 1.5 },
+                { "label": "Q", "x": 1.5, "y": 2.5 },
+                { "label": "W", "x": 2.5, "y": 2.5 },
+                { "label": "E", "x": 3.5, "y": 2.5 },
+                { "label": "R", "x": 4.5, "y": 2.5 },
+                { "label": "T", "x": 5.5, "y": 2.5 },
+                { "label": "Y", "x": 6.5, "y": 2.5 },
+                { "label": "U", "x": 7.5, "y": 2.5 },
+                { "label": "I", "x": 8.5, "y": 2.5 },
+                { "label": "O", "x": 9.5, "y": 2.5 },
+                { "label": "P", "x": 10.5, "y": 2.5 },
+                { "label": "{", "x": 11.5, "y": 2.5 },
+                { "label": "}", "x": 12.5, "y": 2.5 },
+                { "label": "Delete", "x": 15.25, "y": 2.5 },
+                { "label": "End", "x": 16.25, "y": 2.5 },
+                { "label": "PgDn", "x": 17.25, "y": 2.5 },
+                { "label": "Caps Lock", "x": 0, "y": 3.5, "w": 1.75 },
+                { "label": "A", "x": 1.75, "y": 3.5 },
+                { "label": "S", "x": 2.75, "y": 3.5 },
+                { "label": "D", "x": 3.75, "y": 3.5 },
+                { "label": "F", "x": 4.75, "y": 3.5 },
+                { "label": "G", "x": 5.75, "y": 3.5 },
+                { "label": "H", "x": 6.75, "y": 3.5 },
+                { "label": "J", "x": 7.75, "y": 3.5 },
+                { "label": "K", "x": 8.75, "y": 3.5 },
+                { "label": "L", "x": 9.75, "y": 3.5 },
+                { "label": ":", "x": 10.75, "y": 3.5 },
+                { "label": "@", "x": 11.75, "y": 3.5 },
+                { "label": "~", "x": 12.75, "y": 3.5 },
+                {
+                    "label": "Enter",
+                    "x": 13.75,
+                    "y": 2.5,
+                    "ks": [
+                        [0, 0],
+                        [1.5, 0],
+                        [1.5, 2],
+                        [0.25, 2],
+                        [0.25, 1],
+                        [0, 1],
+                        [0, 0]
+                    ]
+                },
```

QMK Configurator's UI code actually doesn't support `ks`, so we do this:

```suggestion
                { "label": "Enter", "x": 13.75, "y": 2.5, "w": 1.25, "h": 2 },
```

... and it renders like an ISO Enter.